src/App.tsx

```diff
@@ -1,14 +1,52 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  const [filteredSelect, setFilteredSelect] = useState('all');
+  const [query, setQuery] = useState('');
+  const [selectedTodo, setSelectedTodo] = useState(0);
+
+  useEffect(() => {
+    getTodos()
+      .then(todos => {
+        setVisibleTodos(todos);
+      });
+  }, []);
+
+  const filteredTodos = visibleTodos.filter(todo => {
+    switch (filteredSelect) {
+      case 'active':
+        return !todo.completed;
+      case 'completed':
+        return todo.completed;
+      case 'all':
+        return todo;
+      default:
+        return todo;
+    }
+  });
+
+  const searchedTodos = filteredTodos.filter(todo => {
+    const titleToLowerCase = todo.title.toLowerCase();
+    const queryToLowerCase = query.toLowerCase();
+
+    return titleToLowerCase.includes(queryToLowerCase);
+  });
+
+  const todosId = (todoId: number) => {
```

Comment:
todosId variable should store some number
now it stores the function, check variable name